packages/bits/src/constants/wizard.constans.ts

```diff
@@ -0,0 +1,11 @@
+import {InjectionToken} from "@angular/core";
+import {IWizardIcons} from "../lib/wizard-v2/wizard.directive";
+
+export const WIZARD_ICONS_PRESET_TOKEN = new InjectionToken<Partial<IWizardIcons>>("WIZARD_ICONS_PRESET");
+
+export const WIZARD_ICONS_PRESET: IWizardIcons = {
+    initial: "step",
+    visited: "step-complete",
+    selected: "step-active",
```

Comment:
Maybe we could use the same vernacular as the icon names?

```suggestion
    complete: "step-complete",
    active: "step-active",
```